examples/config.js

```diff
@@ -18,27 +18,94 @@ module.exports = {
     // trRoutingCacheAllScenarios: false,
     // Configuration for the trRouting services. Single is for the single calculation instance (from the UI and public API), while batch is for the batch calculation instance, for tasks
     // routing: {
-    //   transit: {
-    //     defaultEngine: 'trRouting',
-    //     engines: {
-    //       trRouting: {
-    //         single: {
-    //           port: 4000,
-    //           // Enable caching of connections for all scenarios in trRouting. Will use more memory
-    //           cacheAllScenarios: false,
-    //           debug: false,
-    //           logs: { maxFileSizeKB: 5120, nbFiles: 3 }
-    //         },
-    //         batch: {
-    //           port: 14000,
-    //           // Enable caching of connections for all scenarios in trRouting. Will use more memory and may not be necessary for batch calculations as currently there's only one scenario per task
-    //           cacheAllScenarios: false,
-    //           debug: false,
-    //           logs: { maxFileSizeKB: 5120, nbFiles: 3 }
+    //     transit: {
+    //         defaultEngine: 'trRouting',
+    //         engines: {
+    //             trRouting: {
+    //                 single: {
+    //                     port: 4000,
+    //                     // Enable caching of connections for all scenarios in trRouting. Will use more memory
+    //                     cacheAllScenarios: false,
+    //                     debug: false,
+    //                     logs: { maxFileSizeKB: 5120, nbFiles: 3 }
+    //                 },
+    //                 batch: {
+    //                     port: 14000,
+    //                     // Enable caching of connections for all scenarios in trRouting. Will use more memory and may not be necessary for batch calculations as currently there's only one scenario per task
+    //                     cacheAllScenarios: false,
+    //                     debug: false,
+    //                     logs: { maxFileSizeKB: 5120, nbFiles: 3 }
+    //                 }
+    //             }
     //         }
-    //       }
-    //     }
-    //   }
+    //     },
+    //     // Configuration for the osrm modes 
```

Comment:
Maybe just change the comment here to say something like config for simple routing modes (using engines like OSRM)